plugins/woocommerce-admin/client/task-lists/fills/shipping/index.js

```diff
@@ -687,6 +672,11 @@ const ShippingWrapper = compose(
 		const countryName = country ? country.name : null;
 		const activePlugins = getActivePlugins();
 
+		const prefetchedJetpackAuthData = prefetchJetpackAuthData(
```

Comment:
Prefetching the Jetpack auth URL here and then passing it to the `<Connect>` component so users don't have to wait for the "Connect" to finish retrieving the auth URL from the REST API.